database/sqlite/sqlite_functions.h

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#ifndef NETDATA_SQLITE_FUNCTIONS_H
+#define NETDATA_SQLITE_FUNCTIONS_H
+
+#include "../../daemon/common.h"
+#include "sqlite3.h"
+
+
+/*
+ * Initialize a database
+ */
+
+#define SQLITE_WAL_SIZE (8 * 1024 * 1024 * 1024)
```

Comment:
This is unused, but if we ever use it, it can't be 8GiB :)

Reply:
will remove